test/common_nn.py

```diff
@@ -3136,6 +3149,7 @@ def __init__(self, *args, **kwargs):
         self.FIXME_no_cuda_gradgrad_comparison = \
             kwargs.get('FIXME_no_cuda_gradgrad_comparison', False)
         self.precision = kwargs.get('precision', 2e-4)
+        self.check_reference_only = kwargs.get('check_reference_only', False)
```

Comment:
just reuse the names in CriterionTest -- so this would be check_forward_only.

Reply:
Done